staging/src/k8s.io/apimachinery/pkg/runtime/doc.go

```diff
@@ -18,20 +18,27 @@ limitations under the License.
 // that follow the kubernetes API object conventions, which are:
 //
 // 0. Your API objects have a common metadata struct member, TypeMeta.
+//
 // 1. Your code refers to an internal set of API objects.
+//
```

Comment:
This looks fine to me, but I'm just curious if there is a way to test godoc changes locally?

Reply:
oh, that was easy! :)